### DISCUSSION TITLE
Issues with Hasura iOS documentation?

### DISCUSSION THREAD

**Discussion by bhagyash007** [hasura/graphql-engine#8678] [General] [answered]
https://hasura.io/learn/graphql/ios/apollo-client/ 

Forgot to mention how to download schema using admin key. Was haunted by this " Error: x-hasura-admin-secret/x-hasura-access-key required, but not found " after entering the correct key

Just getting started with Hasura and ended up wasting an hour to finally figure out that the issue comes from incorrect/ missing  documentation

The only snippet shared
apollo schema:download --endpoint=http://hasura.io/learn/graphql --header="Authorization: Bearer <token>" 

The missing snippet and lack of explanation
apollo schema:download --endpoint=http://hasura.io/learn/graphql --header="X-Hasura-Admin-Secret: key"

You guys should build your own SDK instead of making people use Apollo.  Firebase is so good with SDKs

**Comment by adas98012** [comment, marked as answer]
Hi,
   Thanks for your feedback. We currently don't have plan to build our own SDK yet, but that could change if our Product team determines there is a growing need for it.
